Phat-Taupe-Pests/BangazonAPI
# Rock update initializer

## DESCRIPTION
Updated the DBInitializer to add products linked to orders in the ProductOrder table.

## STEPS TO TEST
1. Clone this branch to a local repo
2. In terminal, run `dotnet ef migrations add {migrationName}`, `dotnet ef database update`, `dotnet build` then `dotnet run`
3.  Check the database in DB Browser to see if the ProductOrder table is populated
## REFERENCE

No ticket, just cleaning up our testing process

[ERD](https://github.com/Phat-Taupe-Pests/Bangazon)
